src/OpenLoco/Vehicles/VehicleManager.cpp

```diff
@@ -13,6 +13,26 @@ using namespace OpenLoco::Interop;
 
 namespace OpenLoco::VehicleManager
 {
+    // something to do with animation/rotation of vehicle in build window???
```

Comment:
Although these are vehicle related they are ui only. I feel they should be under a ui manager rather than a vehicle one. When we eventually split the ui from the game we wouldn't want this on the game side.

Reply:
how about WindowManager? otherwise there is Map::AnimationManager, but nothing else seems appropriate

Reply:
Yeah window manager would probably be fine.

Reply:
done